Source/Common/Util.cpp

```diff
@@ -107,6 +107,7 @@ bool pjutil::TerminatedExistingExe()
                     if (TerminateProcess(hHandle, 0))
                     {
                         bTerminated = true;
+                        WaitForSingleObject(hHandle, 30 * 1000);
```

Comment:
This looks like highly inefficient code.

It would be 30 times more optimized to just say `WaitForSingleObject(hHandle, 30000);`.